client/src/components/TermsOfAgreementModal.js

```diff
@@ -31,6 +31,9 @@ const getStudentTerms = () => {
         As a student, you must acknowledge that you are over 13 years of age at the time of registration. One Foot In is not open to students under the age of 13.   
         </p>
         <p>
+        Additionally, we do not by any means represent any schools or universities themselves, but rather provide a context in which members of these communities can interact. 
```

Comment:
For first part: `Additionally, we do not by any means represent any schools or universities themselves` --> `We do not represent schools or colleges themselves` to make the disclaimer sound a little less defensive? Let me know your thoughts